alibi_detect/saving/saving.py

```diff
@@ -188,6 +189,11 @@ def _save_detector_config(detector: ConfigurableDetector,
     if optimizer is not None:
         cfg['optimizer'] = _save_optimizer_config(optimizer)
 
+    # Serialize device
+    device = cfg.get('device')
+    if device is not None:
```

Comment:
Ref

Reply:
Instead of the `_save_device_config` wrapper, isn't it easier just to do `cfg['device'] = save_device_config_pt(device)` here?

Reply:
p.s. Maybe `_save_device` would be more accurate than `_save_device_config`? `_save_optimizer_config` etc are named `_config` since they do actually return a "config `dict`", whereas `_save_device_config` is only returning a `str`.